tasks/upload.rb

```diff
@@ -1,20 +1,31 @@
 #!/opt/puppetlabs/puppet/bin/ruby
 # frozen_string_literal: true
 
+require 'archive/tar/minitar'
```

Comment:
This doesn't always work, puppet-agent doesn't ship with minitar everywhere. We need to use the Puppet helper.